parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -89,15 +89,15 @@ updateAptWithMicrosoftPkg() {
 
 disableNtpAndTimesyncdInstallChrony() {
     # Disable systemd-timesyncd
-    sudo systemctl stop systemd-timesyncd
-    sudo systemctl disable systemd-timesyncd
+    systemctl_stop 20 30 120 systemd-timesyncd || exit $ERR_STOP_SYSTEMD_TIMESYNCD_TIMEOUT
```

Comment:
actually there's a func systemctlDisableAndStop that could do these two steps together

Reply:
checking.

Reply:
systemctlDisableAndStop  doesn't error out. It needs more change

Reply:
nvm on this